index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Brenda Lutiku Portfolio</title>
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css">
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+<header>
+<nav>
+<input type="checkbox" id="check">
+<label for="check"class="checkbtn">
+<i class="fas fa-bars"></i>
+</label>
+<div class="logo">My Logo</div>
+<ul>
+<li><a href ="">Portfolio</a></li>
+<li><a href ="">About</a></li>
+<li><a href ="">Contact</a></li>
+</ul>
+</nav>
+</header> 
```

Comment:
- [ ] Kindly make sure your navigation bar is aligned correctly in order to match the provided Figma design as much as possible. I highly recommend aligning the navigation contents using `flexbox` which gives you a lot of options to align all our elements, plus, it is very easy to manage the app responsiveness.


    - ### YOUR DESIGN
    
    ![image](https://user-images.githubusercontent.com/87197412/234326713-6d6f1355-ed95-4117-be56-f5364a8107aa.png)

    - ### FIGMA DESIGN
    
    ![image](https://user-images.githubusercontent.com/87197412/234326601-fae495ec-e721-4b62-9d99-499bfb53a09c.png)